### PR TITLE
Remove unused /admin/waitinglist code

### LINKED ISSUE #9062: Remove `/admin/waitinglists` page
### Problem

https://openlibrary.org/admin/waitinglists frequently times out because there's too many records and too many queries per record.

The page is presumably supposed to return a list of all books with waitinglists.

Check with @seabelis if this page is needed as I know we have a separate dashboard for Archive.org

image

One workaround is limiting the number of books returned, which has been tested by #9059 and works on testing

### Reproducing the bug

1. Go to ...
2. Do ...

* Expected behavior:
* Actual behavior:

### Context

- Browser (Chrome, Safari, Firefox, etc):
- OS (Windows, Mac, etc):
- Logged in (Y/N): Y
- Environment (prod, dev, local): prod

### PR DESCRIPTION
**Pull request by pidgezero-one**
Closes #9062

Removes the `/admin/waitinglists` page.

### Technical
I removed code that appeared to become unused after taking out the waitinglists template and the function that renders it.

I did not remove any other code related to waiting lists / loans.

### Testing
I haven't tested this much beyond the screenshots below, I am not sure what other areas of the codebase I would need to check for sanity.

Besides that, `docker compose run --rm home make test` returns the same results for me on master and on this branch.

### Screenshot
Before:
image

After (no waitinglist in page listing):
image
image

### Stakeholders
@jimchamp 

Attribution Disclaimer: By proposing this pull request, I affirm to have made a best-effort and exercised my discretion to make sure relevant sections of this code which substantially leverage code suggestions, code generation, or code snippets from sources (e.g. Stack Overflow, GitHub) have been annotated with basic attribution so reviewers & contributors may have confidence and access to the correct context to evaluate and use this code.

### CODE REVIEW

File: openlibrary/core/waitinglist.py [hunk 1]
```diff
@@ -17,7 +17,6 @@
 import logging
 import web
 from openlibrary.accounts.model import OpenLibraryAccount
-from openlibrary.core import cache
```
Comment (pidgezero-one) [inline openlibrary/core/waitinglist.py:20]: unused import

File: openlibrary/core/waitinglist.py [hunk 2]
```diff
@@ -156,40 +155,6 @@ def update(self, **kw):
         dict.update(self, kw)
 
 
-class Stats:
```
Comment (pidgezero-one) [inline openlibrary/core/waitinglist.py:159]: I could only find this being used as WLStats in waitinglist_admin, and removed it as unused code

**Review by cdrini — APPROVED** [review]
Lgtm, thanks @pidgezero-one ! Tested on testing.openlibrary.org . Could you create a new issue to potentially remove the `waitingloan` table ? I don't see it referenced anywhere outside these lines.

**Comment by pidgezero-one** [comment]
@cdrini Sure thing! thank you!

**Pull request merged**